### PR TITLE
multi-fidelity blog

### PR DESCRIPTION
**Pull request by javiergonzalezh**
*Issue #66 , if available:*

*Description of changes:* I have included the blog for the multi-fidelity component.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: _posts/2016-06-10-multifidelity-emulation.markdown [hunk 1]
```diff
@@ -2,288 +2,97 @@
...
+
+linear_mf_model =  GPyMultiOutputWrapper(gpy_linear_mf_model, num_fidelities, n_optimization_restarts = 5)
+linear_mf_model.optimize()
+```
+
+And that's it! In the tutorials you can see how to train other available models like the one described [[2](#references-on-multi-fidelity-gaussian-processes)]. 
+which assumes a non-linear relationship between fidelities. Note that you are now ready to use this model in other decision 
+methods implemented in Emukit.
+
+Check our list of [notebooks]() if you want to learn more about how to do Multi-fidelity emulation with with Emukit. You can also check the Emukit [documentation]().
```
Comment (apaleyes) [inline _posts/2016-06-10-multifidelity-emulation.markdown]: Let's complete these links, the necessary resources are already there

File: _posts/2016-06-10-multifidelity-emulation.markdown [hunk 2]
```diff
@@ -2,288 +2,97 @@
...
+outer-loop applications. To work with an specific example we start loading a problem with two fidelities in which their relationship 
+between is given by $$f_{high}(x) = (x-\sqrt{2})f^2_{low}$$. You can check 
+[[2](#references-on-multi-fidelity-gaussian-processes)] for a graphical representation of this problem. We will use the interval [0,1]
+as the input domain in our experiment.
+
+```python
+from emukit.emukit.test_functions.non_linear_sin import nonlinear_sin_high, nonlinear_sin_low
+n_low_fidelity_points = 50
+
+x_train_l = nonlinear_sin_high(np.linspace(0, 1, n_low_fidelity_points)[:, None])
```
Comment (apaleyes) [inline _posts/2016-06-10-multifidelity-emulation.markdown]: Aren't these low and high backwards?

File: _posts/2016-06-10-multifidelity-emulation.markdown [hunk 3]
```diff
@@ -2,288 +2,97 @@
...
+```python
+gpy_linear_mf_model.mixed_noise.Gaussian_noise.fix(0)
+gpy_linear_mf_model.mixed_noise.Gaussian_noise_1.fix(0)
+```
+and complete the training by writing 
+
+```python
+from emukit.model_wrappers.gpy_model_wrappers import GPyMultiOutputWrapper
+
+linear_mf_model =  GPyMultiOutputWrapper(gpy_linear_mf_model, num_fidelities, n_optimization_restarts = 5)
```
Comment (apaleyes) [inline _posts/2016-06-10-multifidelity-emulation.markdown]: Why would we need a wrapper? You could just call optimize right away. There is no need to use the wrapper unless the model is used for some Emukit loop.

You could however just call `optimize` on the model here, and then separately mention that one could use `GPyMultiOutputWrapper` if they wanted to use these models with Emukit

File: _posts/2016-06-10-multifidelity-emulation.markdown (hunk 1 above)
Comment (javiergonzalezh) [reply _posts/2016-06-10-multifidelity-emulation.markdown]: done!

File: _posts/2016-06-10-multifidelity-emulation.markdown (hunk 2 above)
Comment (javiergonzalezh) [reply _posts/2016-06-10-multifidelity-emulation.markdown]: Correct! changed!

File: _posts/2016-06-10-multifidelity-emulation.markdown (hunk 3 above)
Comment (javiergonzalezh) [reply _posts/2016-06-10-multifidelity-emulation.markdown]: Good point, done.

File: _posts/2016-06-10-multifidelity-emulation.markdown [hunk 4]
```diff
@@ -2,288 +2,97 @@
...
+from emukit.emukit.test_functions.non_linear_sin import nonlinear_sin_high, nonlinear_sin_low
+n_low_fidelity_points = 50
+
+x_train_l = nonlinear_sin_low(np.linspace(0, 1, n_low_fidelity_points)[:, None])
+x_train_h = nonlinear_sin_high(x_train_l[::4, :])
+
+X_train, Y_train = convert_xy_lists_to_arrays([x_train_l, x_train_h], [y_train_l, y_train_h])
+``` 
+ 
+With this code we have just collected some samples in both fidelities and formatted the data in a way that can be understood by Emukit.
```
[the remark this answers is not in the document]
Comment (javiergonzalezh) [reply _posts/2016-06-10-multifidelity-emulation.markdown]: done

File: _posts/2016-06-10-multifidelity-emulation.markdown [hunk 5]
```diff
@@ -2,288 +2,97 @@
...
+These low-fidelity  observations are then systemically combined with the more accurate (but limited) observations in order to predict 
+the high-fidelity output more effectively. 
+
+Emukit offers implementation of a selection Gaussian process multi-fidelity models that can also be combined with other 
+outer-loop applications. To work with an specific example we start loading a problem with two fidelities in which their relationship 
+between is given by $$f_{high}(x) = (x-\sqrt{2})f^2_{low}$$. You can check 
+[[2](#references-on-multi-fidelity-gaussian-processes)] for a graphical representation of this problem. We will use the interval [0,1]
+as the input domain in our experiment.
+
+```python
```
[the remark this answers is not in the document]
Comment (javiergonzalezh) [reply _posts/2016-06-10-multifidelity-emulation.markdown]: I improved this part. We are loading the forrester function.

**Pull request merged**